LayoutTests/imported/w3c/web-platform-tests/cookie-store/cookieStore_opaque_origin.https-expected.txt

```diff
@@ -1,4 +1,4 @@
 
-PASS cookieStore in non-sandboxed iframe should not throw
+FAIL cookieStore in non-sandboxed iframe should not throw assert_equals: cookieStore ${apiCall} should not throw expected "no exception" but got "TypeError"
```

Comment:
I'm not entirely sure why this change happened yet. I'm looking into it, but still wanted to post the patch for review in the meantime.

Reply:
This is fine.  Seems like a test just getting farther than it used to.